tests/Fixtures/Migrations/Version20210820131819.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Zenstruck\Foundry\Tests\Fixtures\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20210820131819 extends AbstractMigration
```

Comment:
Adding/updating/renaming the entities will change this migration, I know generating these migrations are a pain - sorry about that.